packages/braid-design-system/src/lib/components/TextLink/TextLink.docs.tsx

```diff
@@ -71,6 +80,34 @@ const docs: ComponentDocs = {
           </Text>,
         ),
     },
+    {
+      description: (
+        <Text>
+          By default, an icon will be <Strong>leading</Strong> the label,
+          however this can be changed by providing <Strong>iconPosition</Strong>{' '}
+          of <Strong>trailing</Strong>.
+        </Text>
+      ),
+      Example: () =>
+        source(
+          <Inline space="large" alignY="center">
```

Comment:
This example feels like it should just be with `iconPosition="trailing"`, like the one above it. Instead of two `TextLink`s wrapped in an `Inline`.

Or remove the previous example and just keep this one showing both uses?

Reply:
Yep agree, I'll simplify it to just be the one button demoing `trailing`